### ISSUE TITLE
Architect trade prices go up too fast for smaller villages

### ISSUE THREAD

**Issue by Dogez** [TangoTek/TekTopia-Community#837]
I feel like, after only buying a couple houses, a blacksmith, and 2 mineshafts the prices the architect offers for most buildings goes up too high, because it takes quite a while to get enough of a material to sell to the merchant for a limited amount of emeralds. It makes you feel inclined to go out and raid a natural village or desert temple, when I am aware that is not what you want in this mod.

**Comment by stickrai** [comment]
I agree that the early from scratch gameplay takes a LONG time to get going.

Made a whole suggestion post on early gameplay in general and what could be tweaked, this is definitely a valid point though!

**Comment by IridescentShadow** [comment]
I had the same issue early on. Ended up raiding 3 naturally generated villages to have enough to get going
